root/etc/rc.d/rc.local

```diff
@@ -5,7 +5,13 @@ exportKCL
 
 # setup root password
 if [[ ! -z "$KCL_FDI_ROOTPW" ]]; then
-  echo "root:$KCL_FDI_ROOTPW" | chpasswd >/dev/null
+  # remove possible double and/or single quotes surrounding password string
+  KCL_FDI_ROOTPW=`echo $KCL_FDI_ROOTPW | sed -e "s/^'\(.*\)'$/\1/" -e 's/^"\(.*\)"$/\1/'`
```

Comment:
`git rebase -i HEAD~numberofcommits` , then you'll be prompted and you can squash all your commits by changing all of the 'pick' but the first one to 'squash'